includes/Page_Template_Post_Type.php

```diff
@@ -88,10 +88,11 @@ public function register() {
 					'attributes'               => __( 'Page Template Attributes', 'web-stories' ),
 					'insert_into_item'         => __( 'Insert into page template', 'web-stories' ),
 					'uploaded_to_this_item'    => __( 'Uploaded to this page template', 'web-stories' ),
-					'featured_image'           => __( 'Featured Image', 'web-stories' ),
-					'set_featured_image'       => __( 'Set featured image', 'web-stories' ),
-					'remove_featured_image'    => __( 'Remove featured image', 'web-stories' ),
-					'use_featured_image'       => __( 'Use as featured image', 'web-stories' ),
+					'featured_image'           => _x( 'Featured Image', 'template', 'web-stories' ),
+					'set_featured_image'       => _x( 'Set featured image', 'template', 'web-stories' ),
+					'remove_featured_image'    => _x( 'Remove featured image', 'template', 'web-stories' ),
+					'use_featured_image'       => _x( 'Use as featured image', 'template', 'web-stories' ),
```

Comment:
since we use "story template" as the context for story templates, we should use "page template" as the context for page templates.